core/validation/validate.py

```diff
@@ -271,6 +272,7 @@ def validate_enums(
     :return: A list of InvalidEnumValueFailure objects for any rows with values outside
              the set of valid enum values.
     """
+    workbook[sheet_name]["original_index"] = workbook[sheet_name].index
```

Comment:
No, it will cause an error because for some of our DataFrames the indexes will be replicated (where, for instance, a melt has happened), and  ```invalid_value = sheet[column][row_idx]``` will no longer return an individual value, but instead a series of values.